Set SOURCE_DATE_EPOCH to 946684800 in .zshrc

LenzGr opened the pull request:
Setting `SOURCE_DATE_EPOCH` to `0` had some weird side effects when building Python modules. Setting the date to `946684800` (2001-01-01) in `.zshrc` and removing the duplicate definition `setup-ceph.sh` should fix this.

Signed-off-by: Lenz Grimmer <lenz@grimmer.com>

Exotelis commented:
Helped me ;)

tspmelo commented:
https://github.com/ricardoasmarques/ceph-dev-docker/blob/6650b4a04cd752cf0201a8702b48df441c36dae9/shared/zsh/.zshrc#L73 also exports that var.
I think we only need one, so the other should be removed.

LenzGr commented:
> I think we only need one, so the other should be removed.

Good point, thanks for the hint. I now updated the value in `.zshrc` and removed the duplicate one from `setup-ceph.sh`.

ricardoasmarques commented:
@tspmelo @LenzGr this means that we will have different behaviors when using zsh and when calling `setup-ceph.sh` directly without zsh?

Since using `zsh` is optional, I think we should not rely on `.zshrc`... or am I missing something?

tspmelo commented:
@ricardoasmarques not sure if anything works if you don't use zsh, all configs are applied to it.

LenzGr commented:
> Since using `zsh` is optional, I think we should not rely on `.zshrc`... or am I missing something?

It it really? I wasn't aware of this. But it could of course be set in `.profile` instead, which should be Shell-agnostic.

ricardoasmarques commented:
ok, for know lets assume that zsh is used.
